apps/web/app/api/storage/route.ts

```diff
@@ -0,0 +1,334 @@
+import { authOptions } from "@/app/api/auth/[...nextauth]/authOptions";
+import { hasUserEnvironmentAccess } from "@/lib/api/apiHelper";
+import { getServerSession } from "next-auth";
+import { NextRequest, NextResponse } from "next/server";
+import {
+  getFileFromLocalStorage,
+  getFileFromS3,
+  putFileToLocalStorage,
+  putFileToS3,
+} from "@formbricks/lib/services/storage";
+import path from "path";
+
+const UPLOADS_DIR = path.resolve("./uploads");
```

Comment:
Does it make sense to put this `UPLOADS_DIR` into the `constants.ts` file?

Reply:
yes it does since it will be a constant, but the constants file is also imported in the client code and the `path` module will not be available there and will throw an error @mattinannt

Reply:
@pandeymangg Shubi fixed this. constants file is now server only :-) (checked during build time)